doc/manual/src/SUMMARY.md.in

```diff
@@ -28,6 +28,7 @@
   - [Data Types](language/types.md)
     - [String context](language/string-context.md)
   - [Syntax and semantics](language/syntax.md)
+    - [Identifier](language/identifier.md)
```

Comment:
The plural reads as unnatural to me; should ‘Operators’ and ‘Built-ins’ also be singular? There are several operators and several built-ins in the language; there are also several (an infinite number, but if that's *less* plural than ‘several’ then something is wrong) identifiers in the language.